File: medplum-java/medplum-java-model/src/main/java/com/fhir/ImplementationGuide_Template.java
package com.fhir;

import com.StagedBuilderStyle;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import java.lang.String;
import java.util.List;
import java.util.Optional;
import org.immutables.value.Value;

@Value.Immutable
@StagedBuilderStyle
@JsonDeserialize(
    as = ImmutableImplementationGuide_Template.class
)
@JsonIgnoreProperties(
    ignoreUnknown = true
)
public interface ImplementationGuide_Template {
  Optional<List<Extension>> extension();

  Optional<String> source();

  Optional<String> id();

  Optional<List<Extension>> modifierExtension();

  Optional<String> scope();

  Optional<code> code();

  static ImmutableImplementationGuide_Template.Builder builder() {
    return ImmutableImplementationGuide_Template.builder();
  }
}

File: medplum-typescript/packages/sdk/src/model/fhir/ImplementationGuide_Template.ts
import { code } from "./code";
import { Extension } from "./Extension";

export interface ImplementationGuide_Template {
    id: string | null | undefined;
    extension: Extension[] | null | undefined;
    modifierExtension: Extension[] | null | undefined;
    code: code | null | undefined;
    source: string | null | undefined;
    scope: string | null | undefined;
}
